solutions/src/main/java/ru/sberbank/school/task02/FxConverter.java

```diff
@@ -0,0 +1,47 @@
+package ru.sberbank.school.task02;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.Quote;
+import ru.sberbank.school.task02.util.Symbol;
+import ru.sberbank.school.task02.util.Volume;
+
+public class FxConverter implements FxConversionService {
+
+  private ExternalQuotesService Quotes;
+
+  FxConverter(ExternalQuotesService q) {
+    Quotes = q;
+  }
+
+  @Override
```

Comment:
&#39;method def modifier&#39; has incorrect indentation level 2, expected level should be 4.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1880232587>*